### PR TITLE
Configure travis ci to launch jest test and configure jest in repo #108

### PR DESCRIPTION
**Pull request by Eltom777**
Jest works locally but not with Travis (Check console output). As a result, you can start building your unit tests for each feature. 
I will create an issue to fix the bug with Travis and executing "npm test" and "npm run build"

### CODE REVIEW

File: picturepoint/package.json [hunk 1]
```diff
@@ -22,9 +22,10 @@
     "react-scripts": "3.3.1"
   },
   "scripts": {
+    "preinstall": "npx npm-force-resolutions",
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "jest",
```
Comment (Eltom777) [inline picturepoint/package.json:30]: @a-iacampo  this is where we call Jest to execute all the test

**Comment by Eltom777** [comment]
Test are now working due to this commit 67ec1a0.

**Review by a-iacampo — APPROVED** [review]
Demoed in person and discussed all changes

**Review by digathomas — APPROVED** [review]
Demoed via shared screen video call, works perfect.

**Pull request merged**